.travis.yml

```diff
@@ -44,4 +44,4 @@ before_script:
   - android-wait-for-emulator
 
 script:
-  - ./gradlew connectedCheck -x library:signArchives -PdisablePreDex
```

Comment:
it is not checkstyle change - I've edited this file directly on GitHub and it added newline at the end of file automatically